doc/modules/classes.rst

```diff
@@ -1085,9 +1085,11 @@ Hyper-parameter optimizers
    :template: class.rst
 
    model_selection.GridSearchCV
+   model_selection.GridHalvingSearchCV
```

Comment:
I don't have an opinion on whether ``RandomHalving`` or ``HalvingRandom`` is better.
In the rest of the literature, Successive Halving is always randomized, I think. But I felt it would be useful to have a drop-in replacement for grid-search, which is why we added that as well.

Reply:
I think `HalvingGridSearchCV` and `HalvingRandomSearchCV` are good names.